core/src/main/scala/org/broadinstitute/dsde/rawls/openam/StandardUserInfoDirectives.scala

```diff
@@ -1,23 +1,52 @@
 package org.broadinstitute.dsde.rawls.openam
 
-import org.broadinstitute.dsde.rawls.model.{RawlsUserEmail, RawlsUserSubjectId, UserInfo}
+import org.broadinstitute.dsde.rawls.dataaccess.{HttpSamDAO, SamDAO, SlickDataSource}
+import org.broadinstitute.dsde.rawls.dataaccess.jndi.JndiDirectoryDAO
+import org.broadinstitute.dsde.rawls.model.{RawlsUser, RawlsUserEmail, RawlsUserRef, RawlsUserSubjectId, UserInfo, UserStatus}
 import spray.http.{HttpHeader, OAuth2BearerToken}
 import spray.routing.Directive1
 import spray.routing.Directives._
 
-trait StandardUserInfoDirectives extends UserInfoDirectives {
+import scala.concurrent.Future
 
+trait StandardUserInfoDirectives extends UserInfoDirectives with JndiDirectoryDAO {
```

Comment:
I don't think you should extend `JndiDirectoryDAO` since this is calling Sam.

It seems the only thing `JndiDirectoryDAO` is used for is the implicit `ExecutionContext`. Removing the `with JndiDirectoryDAO` and adding `import scala.concurrent.ExecutionContext.Implicits.global` works for me.